src/pages/home/report/ReportActionItem.tsx

```diff
@@ -671,6 +671,7 @@ function ReportActionItem({
                                 shouldDisableEmojiPicker={
                                     (ReportUtils.chatIncludesConcierge(report) && User.isBlockedFromConcierge(blockedFromConcierge)) || ReportUtils.isArchivedRoom(report)
                                 }
+                                isGroupPolicyReport={!!report?.policyID && report.policyID !== CONST.POLICY.ID_FAKE}
```

Comment:
Should we use [isReportInGroupPolicy](https://github.com/Expensify/App/blob/7512346c2fd5c03882e3f9951b03530d8cbf7972/src/libs/ReportUtils.ts#L926) here? See also
https://github.com/Expensify/App/blob/7512346c2fd5c03882e3f9951b03530d8cbf7972/src/components/Composer/types.ts#L80-L81

Reply:
Since the prop is named `...Group...` I think you're right

Reply:
But in `ReportActionCompose` we set `isGroupPolicyReport` like I did. Which one is correct? 🤔 I think we should have the same solution in both places

https://github.com/Expensify/App/blob/7512346c2fd5c03882e3f9951b03530d8cbf7972/src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx#L349

cc @eh2077

Reply:
Let's use the same solution we did for the ReportActionCompose, the problem with using the [isReportInGroupPolicy](https://github.com/Expensify/App/blob/7512346c2fd5c03882e3f9951b03530d8cbf7972/src/libs/ReportUtils.ts#L926) is that users who are not members of the policy but got invited to the room won't have the policy object in onyx.